src/main/java/frc/robot/subsystems/drivetrain/Chassis.kt

```diff
@@ -356,11 +356,12 @@ object Chassis :
                     FieldCentricFacingAngle.withVelocityX(xOutput).withVelocityY(yOutput)
                 }
             )
-            .until {
-                xController.atGoal() &&
-                    yController.atGoal() &&
-                    FieldCentricFacingAngle.HeadingController.atSetpoint()
-            }
+
+    //            .until {
+    //                xController.atGoal() &&
+    //                    yController.atGoal() &&
+    //                    FieldCentricFacingAngle.HeadingController.atSetpoint()
+    //            }
```

Comment:
Given the amount of jitter (and the ability to chain this in autos), we should keep this. If we need to tune more we can do so as well, but it seems very close as is